frontend/src/components/board/AccessBoardForm.tsx

```diff
@@ -62,13 +81,28 @@ const AccessBoardForm: React.FC<AccessBoardFormProps> = ({ id }) => {
           <Divider />
         </Grid>
         <Grid item xs={12}>
-          <TextField
-            label="Password"
-            type="password"
-            helperText={errors.password?.message}
-            error={Boolean(errors.password)}
-            {...register("password")}
-          />
+          <FormControl>
+            <InputLabel htmlFor="outlined-adornment-password">Password</InputLabel>
+            <OutlinedInput
+              label="Password"
+              type={showPassword ? "Text" : "Password"}
+              error={Boolean(errors.password)}
+              {...register("password")}
+              endAdornment={
+                <InputAdornment position="end">
+                  <IconButton
+                    onClick={handleClickShowPassword}
+                    onMouseDown={handleMouseDownPassword}
+                    onMouseUp={handleMouseUpPassword}
+                    edge="end"
+                  >
+                    {showPassword ? <VisibilityOff /> : <Visibility />}
+                  </IconButton>
+                </InputAdornment>
+              }
+            />
+            <FormHelperText>{errors.password?.message}</FormHelperText>
+          </FormControl>
```

Comment:
Tämä on käytännössä identtinen ton BoardCreationForm:in password fieldin kanssa, niin kannattaisi vaan tehdä standalone PasswordField componentti, ja käyttää sitä molemmissa. Esim. toi showPassword usestate voi sitten olla sen sisällä, ja voi saada ton register() funktion ja errorit propseina. Ainut ero noiden kahden välillä taitaa olla, että tää AccesBoardForm voi saada noita erroreita, mutta niitä ei edes taideta käyttää, niin sen voi poistaa? Eks kaikki login-errorit tule vaan alerttina, eikä tekstinä fieldin alla.